setup.cfg

```diff
@@ -28,7 +28,7 @@ tag_prefix = v
 parentdir_prefix = transformers4rec-
 
 [codespell]
-skip = .*pb2.py,./.git,./.github,./.mypy_cache,./dist,./build,./docs/build,.*egg-info.*,versioneer.py,*.csv,*.parquet
+skip = .*pb2.py,.git/*,.github/*,./.mypy_cache,./dist,./build,./docs/build,.*egg-info.*,versioneer.py,*.csv,*.parquet
```

Comment:
`./.github` wasn't working to exclude this directory. Changed to `.github/*` and it works to ignore